examples/bri-3/src/bri/identity/bpiSubjects/agents/bpiSubjectsStorage.agent.ts

```diff
@@ -44,6 +45,7 @@ export class BpiSubjectStorageAgent extends PrismaService {
   }
 
   async createNewBpiSubject(bpiSubject: BpiSubject): Promise<BpiSubject> {
+    bpiSubject.publicKey = bpiSubject.publicKey.toLowerCase();
```

Comment:
no need to store as lower case, only lower case both on places we compare

Reply:
as a general observation, ethers and other library returns us the lowercase MM public key.